apps/src/code-studio/reporting.js

```diff
@@ -281,10 +281,10 @@ reporting.sendReport = function(report) {
       postMilestone = true;
       break;
     case PostMilestoneMode.successful_runs_and_final_level_only:
-      postMilestone = report.pass || appOptions.level.final_level;
+      postMilestone = report.pass || appOptions.level.lastLevelInLesson;
```

Comment:
I think the previous behavior was equivalent to checking `lastLevelInScript`, want to double-check that this change in behavior is intentional.  (If it is intentional, are we using lastLevelInScript anywhere?)

Reply:
You're right, `appOptions.level.final_level` is not equivalent to `appOptions.level.lastLevelInLesson`, let me investigate

Reply:
Ok, I've updated it to check `lastLevelInScript`, that seemed to be equivalent to the previous logic and the intention. Good catch!